### PR TITLE
[Tests-Only]Convert shareRecipient test to pact.io

### PR DESCRIPTION
**Pull request by jasson99**
##Related Issue:
- https://github.com/owncloud/owncloud-sdk/issues/500

### CODE REVIEW

File: tests/shareRecipientTest.js [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
   // TESTING CONFIGS
-  var testUserPassword = 'password'
-  var testUser = 'testUser' + timeRightNow
-  var testGroup = 'testGroup' + timeRightNow
+  var testUserPassword = config.testUserPassword
+  var testUser = config.testUser
+  var testGroup = config.testGroup
 
   beforeEach(function (done) {
     oc = new OwnCloud({
```
Comment (HariBhandari07) [inline tests/shareRecipientTest.js:15]: While you are on this file change all the `var`  to `const`

**Review by HariBhandari07 — COMMENTED** [review]
Also, we need to discuss about this PR.

File: tests/shareRecipientTest.js (hunk 1 above)
Comment (jasson99) [reply tests/shareRecipientTest.js]: fixed

File: tests/shareRecipientTest.js [hunk 2]
```diff
@@ -1,16 +1,15 @@
-describe('Main: Currently testing share recipient,', function () {
+fdescribe('Main: Currently testing share recipient,', function () {
   // CURRENT TIME
-  var timeRightNow = Math.random().toString(36).substr(2, 9)
   var OwnCloud = require('../src/owncloud')
   var config = require('./config/config.json')
 
   // LIBRARY INSTANCE
   var oc
```
Comment (HariBhandari07) [inline tests/shareRecipientTest.js:7]: Also, change these `var` s to `const`.

Comment (jasson99) [reply tests/shareRecipientTest.js]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**